doc/running_local_cluster.md

```diff
@@ -518,7 +516,7 @@ db-sync:
 - `cardano-node/state-cluster0/dbsync.stderr`
 
 
-## Accessing `db-sync` database
+## Accessing the `DB-sync` database
```

Comment:
Same here: 'db-sync' is fine in such a tech context. We can also use 'DB Sync' instead